packages/app/src/debug/DebugCancelledAlert.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <Alert
+    :title="t('debugPage.manuallyCanceled')"
```

Comment:
Just realized I need to decide between `cancelled`/`canceled`

Reply:
Decided on `Cancelled` based on this [styleguide](https://docs.google.com/document/d/1pOZ0Ntm4b1Eev_iY5vLBPhOxx66Z8Njrvvg6WMCvWRI/edit#heading=h.daxa2xpyrsop)